en/drafts/originals/text-mining-youtube-comments.md

```diff
@@ -17,115 +17,114 @@ review-ticket: https://github.com/programminghistorian/ph-submissions/issues/374
+This lesson will guide you through three key steps for 1) data collection, 2) cleaning and modeling, and 3) analysis and visualization. 
 
-Secondly, this lesson introduces how to use R to pre-process and clean YouTube comment data and associated YouTube video metadata.
+First, this lesson overviews the preparatory steps for gathering data, including considering ethical issues related to downloading and analyzing YouTube data, as well as the basics of installing R and RStudio, and using the open-source [YouTube Data Tools](https://ytdt.digitalmethods.net/) software. The [Data Collection section](#data-collection) explains how to access and query YouTube’s Application Programming Interface (API) with the user-friendly YouTube Data Tools to download video metadata and comments. A brief discussion explains how to use YouTube Data Tools to search for and download video comments as tabular data (`.csv` file) for further manual or computational analysis with and beyond R. 
 
-Thirdly, this lesson will teach you how to model the YouTube comment data with the Wordfish algorithm using Ken Benoit's [`quanteda`](https://tutorials.quanteda.io/machine-learning/Wordfish/) R package. This [Analysis and Visualisation section](link-to-section) demonstrates how to computationally analyze the comment data in R using Wordfish, an [unsupervised text-scaling algorithm](link-to-definition) for interpreting the primary dimensions of latent meaning within bodies of text data.
+Secondly, this lesson introduces how to use R to pre-process and clean YouTube comment data, as well as associated video metadata.
 
-## YouTube and Scholarly Research
-YouTube houses a wealth of culturally-relevant data that researchers and academics have begun to explore. While YouTube has often been associated with entertainment, it is also a place where significant debates, explicit and implicit, play out between diverse demographics across the political spectrum. 
+Thirdly, this lesson will teach you how to model YouTube comment data with the Wordfish algorithm, using Ken Benoit's [`quanteda`](https://tutorials.quanteda.io/machine-learning/Wordfish/) R package. This [Analysis and Visualisation section](link-to-section) demonstrates how to computationally analyze the comment data in R using Wordfish.

```

Comment:
I believe the section headings may have changed since this was written – which section would you like to link to instead here?

Reply:
Let's say "The [Modeling](link) and  [Visualization](link) sections demonstrate..."